### PR TITLE
New repo structure

### PR DESCRIPTION
**Pull request by KirillTim**
I decide to fix our project structure, like it was suggested [here](http://python-guide-pt-br.readthedocs.io/en/latest/writing/structure/)
Licence is now located in separate file, I get the copy from [here](https://github.com/mozilla/kuma/blob/master/LICENSE). It is very easy to forget to add the licence comment in the file header(like I do for the all files in https://github.com/marco-c/crashsimilarity/pull/66)
py.test is cool. Mozilla [use](https://agopian.info/presentations/2015_06_djangocon_europe/) it.
In the future `docs/` and `setup.py` will be add(when they will be needed).

P.S: Lets drop python2.7 support https://github.com/marco-c/crashsimilarity/issues/69 ?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+init:
+	pip install -r requirements.txt
```
Comment (marco-c) [inline Makefile]: Can you also install test-requirements.txt?

**Comment by marco-c** [comment]
There are conflicts to resolve, can you pull from master?

File: Makefile (hunk 1 above)
Comment (KirillTim) [inline Makefile]: Done

**Comment by KirillTim** [comment]
> There are conflicts to resolve, can you pull from master?

Done

**Comment by marco-c** [comment]
You also need to fix the imports in tests/test_signature_coherence.py, tests/test_signatures_similarity.py and tests/test_top_similar_traces.py

File: tests/.cache/v/cache/lastfailed [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (marco-c) [inline tests/.cache/v/cache/lastfailed]: I think this file shouldn't be added to the repo

**Comment by KirillTim** [comment]
My bad.
Nothing good will happened if merge in hurry.
Will fix in ~5 hours, this evening

**Comment by KirillTim** [comment]
It seems like all is OK now.
BTW, a little bit funny to fix files that will be removed in next PR :)

**Comment by marco-c** [comment]
Thanks!

> BTW, a little bit funny to fix files that will be removed in next PR :)

Indeed :D, but we need to keep the tests green at all times.

**Pull request merged**